### PR TITLE
Wallet-click_CU-md8hjb

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-md8hjb Wallet click](https://app.clickup.com/t/md8hjb)

**Comment by lpboucher** [comment]
Task linked: [CU-md8hjb Wallet click](https://app.clickup.com/t/md8hjb)

**Comment by a-wilkes** [comment]
Same question as with [Cart-click](https://github.com/lpboucher/dak-coffee-wholesale/pull/45#issuecomment-860589252).

File: src/app/shared/components/wallet-modal/wallet-modal.component.ts
Comment (lpboucher) [inline src/app/shared/components/wallet-modal/wallet-modal.component.ts]: minor change: prefer != null, the != checks for null and undefined

File: src/app/shared/components/wallet-modal/wallet-modal.component.ts
Comment (lpboucher) [inline src/app/shared/components/wallet-modal/wallet-modal.component.ts]: if a union type for undefined is passed there is no need to define an initial value

**Review by lpboucher — CHANGES_REQUESTED** [review]
This is approved. Will wait until the refactor is merged to merge this

**Pull request merged**
